cypress/e2e/right-panel/file-panel.spec.ts

```diff
@@ -134,7 +134,9 @@ describe("FilePanel", () => {
                 });
 
                 // Exclude timestamps and read markers from snapshot
-                const percyCSS = ".mx_MessageTimestamp, .mx_RoomView_myReadMarker { visibility: hidden !important; }";
+                // FIXME: hide mx_SeekBar because flaky - see https://github.com/vector-im/element-web/issues/24897
+                const percyCSS =
+                    ".mx_MessageTimestamp, .mx_RoomView_myReadMarker, .mx_SeekBar { visibility: hidden !important; }";
```

Comment:
Hi @luixxiul, thank you for this!

If my understanding is correct, we should remove this once https://github.com/vector-im/element-web/issues/24898 is fixed? If so, I think it would be helpful to link that issue:

```suggestion
                // FIXME: hide mx_SeekBar because flaky - see https://github.com/vector-im/element-web/issues/24897
                //   Remove this once https://github.com/vector-im/element-web/issues/24898 is fixed.
                const percyCSS =
                    ".mx_MessageTimestamp, .mx_RoomView_myReadMarker, .mx_SeekBar { visibility: hidden !important; }";
```

Reply:
That is indeed correct, thanks for the suggestion :-)